lib/tasks/karma.rake

```diff
@@ -1,14 +1,22 @@
 namespace :karma  do
-  task :start => :environment do
+  task :start => :environment do |task|
+    continue_only_in_test_env task
     with_tmp_config :start
   end
 
-  task :run => :environment do
+  task :run => :environment do |task|
+    continue_only_in_test_env task
     with_tmp_config :start, "--single-run"
   end
 
   private
 
+  def continue_only_in_test_env task
+    if Rails.env != 'test'
+      raise "Task must be called in test environment:\n  bundle exec rake #{task.name} RAILS_ENV=test"
```

Comment:
Good. That does the job. Do you think the stack trace could be confusing for people? Devs should be okay, it's just a bit of clutter.